groups/cdn/profiles/development-eu-west-2/vars

```diff
@@ -22,3 +22,7 @@ environments = [
   "titans1",
   "toro1"
 ]
+
+cors_allowed_origins = [
+  "https://*.aws.chdev.org"
+]
```

Comment:
Check your IDE or editor to see if there's an option to enforce trailing newline characters when saving files—both `git` and GitHub will continue to display warnings (as seen here) for files that don't contain one. Update all three `vars` files to add the missing trailing newline.

Reply:
Good shout, sorted 👍🏻